frontend/src/metabase/services.js

```diff
@@ -202,12 +202,6 @@ export const AutoApi = {
   }),
 };
 
-export const EmailApi = {
```

Comment:
I guess I can just delete this now? We don't need this for the public api or anything?